CHANGELOG_PENDING.md

```diff
@@ -11,6 +11,11 @@
 
 ### Bug Fixes
 
-- [codegen/nodejs] - Generate an install script that runs `pulumi plugin install` with 
+- [sdk/python] - Prevent `ResourceOptions.merge` from promoting between the
+  `.provider` and `.providers` fields. This is a subtle breaking change. This changes
```

Comment:
These are user facing docs. I'm wondering if we can't describe coherently a scenario that's breaking and what the user should do should we just omit the words "subtle breaking change"? Can continue presenting it as a bugfix for the interaction of the two fields (that is treat current behavior prior to fix as an error).